src/mainfinal.js

```diff
@@ -0,0 +1,71 @@
+import {
+  guardarReceta,
+ conseguirRecetas,
+ alConseguirRecetas,
+ borrarReceta,
+ conseguirReceta,
+ actualizarReceta,
+}
+  from "./lib/firestore.js";
+
+const formPublicacion = document.getElementById('formPublicacion');
+const postPublicado = document.getElementById('postPublicado');
+let editandoReceta = false;
+let id = '';
+
+window.addEventListener('DOMContentLoaded', async () => {
+  alConseguirRecetas((querySnapshot) => {
+    let html = '';
+    querySnapshot.forEach((doc) => {
+      const publicacion = doc.data();
+      html += `
```

Comment:
Consistencia, si están usando nodos, sigan por el camino de crear nodos, en caso de que quieran cambiar a Plantillas literales entonces como es el caso de la variable html, entonces seria bueno que pudiesen hacer el cambio en todos los componentes (nada recomendable) pero si no es el camino, entonces esto debería seguir siendo nodos.
```suggestion
      html += `
```